manifests/repo.pp

```diff
@@ -34,7 +34,13 @@
       Foreman::Repos['foreman'] -> Package['centos-release-scl-rh']
     }
 
-    if $facts['os']['release']['major'] == '8' and ($repo == 'nightly' or versioncmp($repo, '2.5') >= 0) {
+    if $facts['os']['release']['major'] == '8' and ($repo == 'nightly' or versioncmp($repo, '3.2') >= 0) {
+      package { 'foreman:latest':
```

Comment:
yes, this essentially breaks the fix from #1015, but I need to see if it works before we rename the stream on `yum.tfm`